catalyst/rl/offpolicy/scripts/run_discrete_samplers.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+
+import os
+import copy
+import atexit
+import argparse
+from pprint import pprint
+import multiprocessing as mp
+from redis import StrictRedis
+import torch
+
+from catalyst.dl.scripts.utils import prepare_modules
+from catalyst.contrib.registry import Registry
+from catalyst.utils.config import parse_args_uargs, save_config
+from catalyst.utils.misc import set_global_seeds, boolean_flag
+from catalyst.rl.offpolicy.discrete_sampler import Sampler
```

Comment:
if the sampler would be general, there is no need for this additional copy-paste

Reply:
run_discrete_samplers.py was deprecated in the last commit